src/Common/Thermodynamics/config_numerical_method.jl

```diff
@@ -82,8 +82,10 @@ function sa_numerical_method(
     _T_min::FT = T_min(param_set)
     q_pt = PhasePartition(q_tot, FT(0), q_tot) # Assume all ice
     T_2 = air_temperature(param_set, e_int, q_pt)
-    T_1 = max(_T_min, air_temperature(param_set, e_int, PhasePartition(q_tot))) # Assume all vapor
-    T_2 = bound_upper_temperature(T_1, T_2)
+    T_1 = air_temperature(param_set, e_int, PhasePartition(q_tot)) # Assume all vapor
```

Comment:
We can also give a rigorous bound here:
```suggestion
    T_1 = T_0 +
           ( e_int - q.tot * e_int_v0 )  / (cv_d + (cv_v - cv_d) * q.tot)  # Assume all vapor for energy, and all liquid for specific heat
```
(This likewise is not easily computed from the thermodynamics functions, because it assumes all vapor for the energy, and all liquid for the specific heat, to get a rigorous lower bound.)